contracts/VaultV3.vy

```diff
@@ -292,11 +298,65 @@ def migrateStrategy(new_strategy: address, old_strategy: address):
 @external
 def updateMaxDebtForStrategy(strategy: address, new_maxDebt: uint256):
     # TODO: permissioned: DEBT_MANAGER
-    assert self.strategies[strategy].activation != 0
+    assert self.strategies[strategy].activation != 0, "inactive strategy"
     # TODO: should we check that totalMaxDebt is not over 100% of assets?
     self.strategies[strategy].maxDebt = new_maxDebt
     # TODO: should this emit an event?
 
+
+@external
+def updateDebt(strategy: address) -> uint256:
+    # TODO: permissioned: DEBT_MANAGER (or maybe open?)
+    # TODO: rebalance debt. if the strategy is allowed to take more debt and the strategy wants that debt, the vault will send more. if the strategy has too much debt, the vault will have less
+    currentDebt: uint256 = self.strategies[strategy].currentDebt
+
+    minDesiredDebt: uint256 = 0
```

Comment:
Is the initialization to zero value required?

Reply:
apparently so. i get this error when i tried to declare without initial assignment

```
vyper.exceptions.VariableDeclarationException: Memory variables must be declared with an initial value
```